build.gradle

```diff
@@ -8,22 +7,25 @@ buildscript {
 
 plugins {
     id 'java'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.21'
+    id 'org.jetbrains.kotlin.jvm' version '1.9.20'
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.9.20"
     id "io.spring.dependency-management" version '1.0.6.RELEASE'
-    id 'com.github.johnrengelman.shadow' version '5.0.0'
+    id("io.ktor.plugin") version "2.3.6"
```

Comment:
Just curious the id() vs id ' ' is for a specific reason?

Reply:
i think i wasn't paying full attention, i just made a commit to be more consistent about the `build.gradle` where everything is without parenthesis